postpic/helper.py

```diff
@@ -507,17 +517,17 @@ def __init__(self, fftsize_max=10000, factors=(2, 3, 5, 7, 11, 13)):
                             for powers
                             in itertools.product(*powers_ranges)  # build all combination of pwrs
                             )
-
         self.fftsizes = np.array(list(sorted(filter(lambda x: x <= fftsize_max, fftsizes))))
 
     def __call__(self, n):
         '''
         In the list of sizes calculated at initialization, find the next good value equal
         or larger than a given `n`.
         '''
-        if n > self.fftsize_max:
-            raise ValueError("Given grid is larger then {}. Please use a custom object of "
-                             "the class helper.FFTW_Pad.".format(self.fftsize_max))
+        if n > self.fftsizes[-1]:
```

Comment:
After fftsize_max was set to 0 (the default), the list fftsizes has length 0 - so this fails with an IndexError.